src/librustc_mir/const_eval.rs

```diff
@@ -527,6 +527,12 @@ pub fn error_to_const_error<'mir, 'tcx>(
     ConstEvalErr { error: error.kind, stacktrace, span: ecx.tcx.span }
 }
 
+pub fn undefined_behavior_error<'mir>() -> String {
```

Comment:
Maybe: 
```suggestion
pub fn note_on_undefined_behavior_error<'mir>() -> String {
```
because it is not actually an error message.